### PR TITLE
Update group_four.md

### PR DESCRIPTION
**Pull request by josh-works**
Our plan for the gear up!

### CODE REVIEW

File: m4_sessions/1706-inning/group_four.md [hunk 1]
```diff
@@ -1,63 +1,77 @@
...
-Feelings and thoughts:
 
-- people feel confident in their ability to contribute to the community
-- people have the knowledge of where to go to contribute
-- the knowledge of HOW to contribute (like OSS PR)
 
+# Session outline:
 
 
+### 1. Do we have a responsibility to "give back"?
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_four.md:21]: Is this a discussion or just something you are posing to them?

File: m4_sessions/1706-inning/group_four.md [hunk 2]
```diff
@@ -1,63 +1,77 @@
...
 
-Ideas:
+### 2. Have attendees make list of all things they know AND all the ways they've been helped by folks in their current mod or mods ahead.  (put these questions on screen) (led by Valary)
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_four.md:23]: I like this as a starting point!

File: m4_sessions/1706-inning/group_four.md [hunk 3]
```diff
@@ -1,63 +1,77 @@
...
-  - PR checklist/template
+---- budgeting ten minutes to this point ----
 
-### Session Outline:
+### 3. watch [video](https://youtu.be/nXIIW8TusWw?t=9m58s) (8 min) (Sergey)
 
+---- now 18 mins in ----
 
-- do we have a responsibility to "give back" to the community?
+### 4. Brainstorm how they can get involved (turn and talk) (5 min) (Sergey)
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_four.md:31]: Turn and Talk!! Yes!

File: m4_sessions/1706-inning/group_four.md [hunk 4]
```diff
@@ -1,63 +1,77 @@
...
+### 6. Now we can share some of our ideas
+  - Making PR in Turing repo (John)
+  - host lunch-time in-mod session
 
-Walk through example of PR against Turing repo to fix problem
+  **To Discuss:**
 
-Walk through example of "student interest group", pairing, be a mentor
+> 1. Talk about how to make a PR against a lesson (gist w/instructions?) (John B) ([demo against this repo](https://github.com/turingschool/front-end-curriculum/blob/gh-pages/projects/ideabox.md))
+> 2. In-person involvement (like TryTuring). You're most qualified to help at a try-turing event after _mod 1_. (Kelley M)
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_four.md:46]: Let me know if you  need the upcoming try turing dates.

File: m4_sessions/1706-inning/group_four.md [hunk 5]
```diff
@@ -1,63 +1,77 @@
...
-- videos
-- list of projects to contribute to
-- how to make an impact at Turing
-  - make PR on lessons
-  - make oss contributions
-  - PR checklist/template
+---- budgeting ten minutes to this point ----
 
-### Session Outline:
+### 3. watch [video](https://youtu.be/nXIIW8TusWw?t=9m58s) (8 min) (Sergey)
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_four.md:27]: Which 8 minutes are you watching?

File: m4_sessions/1706-inning/group_four.md [hunk 6]
```diff
@@ -1,63 +1,77 @@
...
+- leverage your inner newb for good
+- making contributions in an industry where that's rewarded
 
-Feelings and thoughts:
 
-- people feel confident in their ability to contribute to the community
-- people have the knowledge of where to go to contribute
-- the knowledge of HOW to contribute (like OSS PR)
 
+# Session outline:
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_four.md:18]: Might want to just intro the team and also be sure to give them time to do some introductions!

File: m4_sessions/1706-inning/group_four.md [hunk 7]
```diff
@@ -1,63 +1,77 @@
...
+- Josh Thompson
 - Sergey L
 - Valery Trudell
 - Kelly Miller
 - John Binning
-- Josh Thompson
+
+
+
+we're "group 4" from:
```
Comment (emhickmann) [inline m4_sessions/1706-inning/group_four.md:76]: Great job with team!!  @josh-works Let me know if you have any questions on these comments.

**Review by emhickmann — COMMENTED** [review]
@josh-works This session is in a great place.  There are a few comments for you to consider.

File: m4_sessions/1706-inning/group_four.md (hunk 5 above)
Comment (lukyans) [reply m4_sessions/1706-inning/group_four.md:27]: starting from here: https://youtu.be/nXIIW8TusWw?t=9m58s

**Pull request merged**